slack-base/src/main/java/com/hubspot/slack/client/models/blocks/elements/UnknownBlockElement.java

```diff
@@ -0,0 +1,11 @@
+package com.hubspot.slack.client.models.blocks.elements;
+
+public class UnknownBlockElement implements BlockElement {
+  protected UnknownBlockElement() {
+  }
+
+  @Override
+  public String getType() {
+    return "";
```

Comment:
Same as above, I'd really like to see this actually inherit whatever type we get back, when we deserialize things using this